source/ci_source/providers/index.ts

```diff
@@ -78,7 +80,8 @@ const realProviders = [
   CodeBuild,
   Codefresh,
   AppCenter,
-  Cirrus
+  Cirrus,
+  AzureDevops,
 ]
 
```

Comment:
I think bringing just this and the Providers change would get you most bang for least effort, this means Danger will at least run on the CI

Reply:
I'm not sure if I understand that correctly. Do you want me to get rid of any of the changes not related to the Azure Devops Provider?

Reply:
Yeah, that would make it usable, then everything else can be built on top of that PR (instead of of making a mega PR - the `danger init` stuff could then come in as it's own PR too for example